nautobot/dcim/tests/test_filters.py

```diff
@@ -1438,7 +1438,7 @@ def setUpTestData(cls):
         InventoryItem.objects.create(device=devices[2], name="Inventory Item 3", manufacturer=cls.manufacturers[2])
 
     def test_description(self):
-        descriptions = [m.description for m in self.manufacturers[:2]]
+        descriptions = [m.description for m in self.manufacturers[::2]]
```

Comment:
Is this a typo?

Reply:
Intentional. Step slice to get the first item, skip the next one, and then one after that.